File: app/kata.py
# exclusive-or-xor-logical-operator
import codewars_test as test
# -------------------------------- MY SOLUTION ------------------------------- #
def xor(a,b):
  return a != b
# ------------------------------ CLEVER SOLUTION ----------------------------- #
# ----------------------------------- TEST ----------------------------------- #
test.assert_equals(xor(False, False), False, "False xor False == False")
test.assert_equals(xor(True, False), True, "True xor False == True")
test.assert_equals(xor(False, True), True, "False xor True == True")
test.assert_equals(xor(True, True), False, "True xor True == False")

File: app/kata.js
// grasshopper-personalized-message
/* ------------------------------- MY SOLUTION ------------------------------ */
function greet (name, owner) {
  return name == owner ? 'Hello boss' : 'Hello guest'
}
/* ----------------------------- CLEVER SOLUTION ---------------------------- */
/* ---------------------------------- TEST ---------------------------------- */
const chai = require("chai");
const Test = chai.assert.strictEqual; // replace Test.assertEqual with Test
chai.config.truncateThreshold=0;

describe("Tests", () => {
  it("test", () => {
Test(greet('Daniel', 'Daniel'), 'Hello boss')
Test(greet('Greg', 'Daniel'), 'Hello guest')
  });
});
